### PR TITLE
updating license

### PR DESCRIPTION
**Pull request by amontanez24**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2,6 +2,8 @@ MIT License
 
 Copyright (c) 2018, MIT Data To AI Lab
 
+Copyright © 2022 DataCebo, Inc.
```
Comment (npatki) [inline LICENSE:5]: Will the © character properly show up in the license? This is is exactly what the doc has specified, but I guess we should keep the character consistent -- use (c) in both places or © in both places

File: LICENSE [hunk 2]
```diff
@@ -20,3 +22,8 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
```
Comment (npatki) [inline LICENSE:23]: Out of curiosity, did you check to see if the rest of the wording is the same between the current License and the doc? That might be good to validate.

**Review by npatki — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 2 above)
Comment (amontanez24) [reply LICENSE:23]: they're the same

File: LICENSE (hunk 1 above)
Comment (amontanez24) [reply LICENSE:5]: Not sure, but I've seen `(c)` in other libraries so I'll do that one

**Review by npatki — APPROVED** [review]
LGTM!

**Pull request merged**